plugins/goodreads.koplugin/main.lua

```diff
@@ -6,6 +6,7 @@ local InfoMessage = require("ui/widget/infomessage")
 local UIManager = require("ui/uimanager")
 local Screen = require("device").screen
 local _ = require("gettext")
+local NetworkMgr = require("ui/network/manager")
```

Comment:
Probably makes it slightly faster to start up the program for those who don't use goodreads at all. I think the difference is so little that we can simply ignore it ;)